steamtinkerlaunch

```diff
@@ -7427,43 +7467,78 @@ function installd3d47dll {
 	fi
 }
 
+function autoBumpReShade {
+	RSVERSLATEST="$( fetchGitHubTags "$RESHADEPROJURL" "1" )"
+	if [ "$AUTOBUMPRESHADE" -eq 1 ] && [ "$RSVERS" = RSVERSLATEST ]; then
```

Comment:
Ahh I see what you meant in your comment. Should be an easy fix :smile: 

If it's storing `$AUTOBUMPRESHADE` then you'll want to compare with `[[ "$AUTOBUMPRESHADE" = "1" ]]` (note the double square brackets - Bash is fussy about this...)

For the second check, this will fail unless you use double square brackets. So it needs to be `[[ "$RSVERS" = RSVERSLATEST ]]`

This should hopefully resolve the problems :-)